SalesforceReact.podspec

```diff
@@ -0,0 +1,33 @@
+Pod::Spec.new do |s|
```

Comment:
Should this file live under the `iOS` directory?

Reply:
Actually no.
When you point to a git repo from a podfile, you don't get to specify a path. Pod specs are expected to be in the top directory.

Reply:
Sounds good.